package.json

```diff
@@ -38,6 +38,7 @@
         "@aws-cdk/lambda-layer-kubectl-v25": "^2.0.3",
         "@aws-cdk/lambda-layer-kubectl-v26": "^2.0.0",
         "@aws-sdk/client-secrets-manager": "^3.316.0",
+        "@instana/aws-eks-blueprint-addon": "^1.0.4",
```

Comment:
@arpitn2020 Why is this still not removed.?

Reply:
@elamaran11 I think there was some confusion. I thought only eks-blueprint dependency that i've added required removal. i removed that as well.